### ISSUE TITLE
fonts not loading, stays default on dice

### ISSUE THREAD

**Issue by Taiine** [charmaur/PolyDiceGenerator#6]
As a dice maker, I was hoping to use this as a faster and easier means to font my dice than going through the long and hard process through blender. Cheers for making this. However, I am having trouble as some of the fonts I've created for my dice fail to load, and instead, it shows the default font face. 
There's no error log, the text just remains default. 

I'm unsure if it's my end and not typing the name of the font correct or if it's something with the font format. All my fonts are TrueType fonts made in FontForge.

https://i.imgur.com/ZfjAP3x.png
there's the one for an example. 
https://i.imgur.com/6N1Nfkx.png
it loads fine in photoshop

So it would be DNDNumGrot:style=Bold as per how it shows? But this shows the default font instead. even removing the :style=Bold it still won't show it. 

https://i.imgur.com/WkBaFMI.png
the numbers stay default.

Most other fonts I have work fine, but I also see the same thing with a few other fonts that I have. 
any advice on why it's doing this?

**Comment by Taiine** [comment]
note also just downloaded the update I saw that came out two days ago. same issue.

**Comment by charmaur** [comment]
The font list dialog (available under Help > Font List) shows the font name and the font style for each available font. You can copy the font name from that list to make sure you have the correct name. If that doesn't help please send me one of the problem fonts so I can troubleshoot. 

Thanks,
Charles

**Comment by Taiine** [comment]
Thanks for the reply! 

I checked the font list and found the fonts in question wasn't being listed while others were. While seeing what was in the update I noted its said the fonts also need to be in the OpenSCAD\fonts folder? All other fonts I've downloaded and some of the fonts I've made do get listed with out this, but I opted to dump them all in anyways. 

Now they all get listed and work fine so all is good. 
But I don't understand why some didn't need to be in that folder, yet others did? 

Anyway it's all working good now that I added the fonts to that folder. 

Thank you so much for the work on this tool, it's a life saver and I can finally start adding new font options that I've been wanting to with out wasting 3 hours or more in blender.
